ramalama/plugins/runtimes/inference/common.py

```
@@ -237,6 +259,108 @@ def _serve_handler(self, args: argparse.Namespace) -> None:
+
+        ref_dir = os.path.join(root, refs_dir_name)
+        for ref_file_name in os.listdir(ref_dir):
+            ref_file_path = os.path.join(ref_dir, ref_file_name)
+            ref_file = migrate_fn(ref_file_path, os.path.join(root, snapshots_dir_name))
+            if ref_file is None:
+                ref_file = RefJSONFile.from_path(ref_file_path)
+
+            tag, _ = os.path.splitext(ref_file_name)
+            model_rel = root.replace(store.path, "").lstrip(os.sep)
+            parts = model_rel.split(os.sep)
+            readable = "-".join(parts + [tag])
```

Ruff (0.15.15)

[warning] Consider `[*parts, tag]` instead of concatenation

Replace with `[*parts, tag]`

(RUF005)